src/ngraph/op/reduce_prod.cpp

```diff
@@ -0,0 +1,99 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/op/reduce_prod.hpp"
+#include "ngraph/graph_util.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+constexpr NodeTypeInfo op::v1::ReduceProd::type_info;
+
+op::v1::ReduceProd::ReduceProd(const Output<Node>& arg,
+                               const AxisSet& reduction_axes,
```

Comment:
Actually we don't need this constructor due to ReduceProd consumes two inputs according to spec.

Reply:
Done